starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -1232,24 +1232,32 @@ sub check_star_catalog {
 	    # ignore precision errors in color
 	    my $color = sprintf('%.7f', $c->{"GS_BV$i"});
 	    $c->{"GS_NOTES$i"} .= 'c' if ($color eq '0.7000000'); # ACA-033
-            $c->{"GS_NOTES$i"} .= 'C' if ($color eq '1.5000000') && ($mag > 10.0);
+            $c->{"GS_NOTES$i"} .= 'C' if ($color eq '1.5000000');
 	    $c->{"GS_NOTES$i"} .= 'm' if ($c->{"GS_MAGERR$i"} > 99);
 	    $c->{"GS_NOTES$i"} .= 'p' if ($c->{"GS_POSERR$i"} > 399);
 	    $note = sprintf("B-V = %.3f, Mag_Err = %.2f, Pos_Err = %.2f",
                             $c->{"GS_BV$i"}, ($c->{"GS_MAGERR$i"})/100, ($c->{"GS_POSERR$i"})/1000)
                 if ($c->{"GS_NOTES$i"} =~ /[Ccmp]/);
 	    $marginal_note = sprintf("$alarm [%2d] Marginal star. %s\n",$i,$note) if ($c->{"GS_NOTES$i"} =~ /[^b]/);
-            # for B-V = 0.7 and guide, red warnings
-            # for all others, including (B-V = 1.5 and guide), yellow warning
-            if ( $marginal_note ){
-                if ($color eq '0.7000000' && $type =~ /BOT|GUI/ ) { 
+            # for B-V = 0.7 and orange warnings
```

Comment:
When updating code comments, bring it to modern standards.  Capitalized and somewhat sentence-like if it improves readability.  In this case
```
# For B-V = 0.7 print orange warning.
# For all others, ..., print yellow warning.
```